grudge/symbolic/operators.py

```diff
@@ -486,7 +486,7 @@ def __init__(self, dd_in=None, dd_out=None):
         if dd_in is None:
             dd_in = prim.DD_VOLUME
         if dd_out is None:
-            dd_out = dd_in
+            dd_out = prim.DD_VOLUME
```

Comment:
@inducer This change broke a few things. I fixed the ones that were causing test failures, but I'm not positive that I got everything. Is there anything else I should check to be more confident that this is ok?

Reply:
IMO: Tests are what provides confidence. If it wasn't tested, you may assume it was broken and don't need to worry about it.